src/PatientChecking/PatientChecking/Controllers/PatientController.cs

```diff
@@ -16,11 +20,15 @@ public class PatientController : BaseController
 
         private readonly IPatientService _patientService;
         private readonly IAppConfigurationService _provinceCityService;
+        private readonly INotyfService _notyf;
```

Comment:
Rename it to `_notyfService`

Reply:
I have removed this since I'm not going to use it anymore. Please review this @vietvo361